src/components/static.tsx

```diff
@@ -46,6 +48,15 @@ const VerificationPage = memo(() => (
           <Card.Title className="text-center">
             Please verify your email account before proceeding 🪪
           </Card.Title>
+          <Button
```

Comment:
This component is no longer relevant. Use the one in navigation.js.

Reply:
Changed to navigation